mosaic_creator.py

```diff
@@ -92,10 +126,27 @@ def createPhotomosaic(target_image, input_images, grid_size,
     mosaic_image = createImageGrid(output_images, grid_size)
     return (mosaic_image)
 
+try:
+    target_image = Image.open(args.target)
+except FileNotFoundError:
+    print("Can not find the specified --target file. Please, make sure the passed file exists.")
+    sys.exit()
+except ValueError:
+	print("Invalid parameters passed to PIL.Image.open() method.")
+	sys.exit()
+except UnidentifiedImageError:
+	print("Can not identify and open the image file specified. Please, make sure the file passed is a valid image.")
+	sys.exit()
+except IsADirectoryError:
+    print("The argument passed as --target is a directory without any files. Image file path expected.")
+    sys.exit()
 
-target_image = Image.open(args.target)
 # input images
-input_images = getImages(args.images)
+try:
+    input_images = getImages(args.images)
```

Comment:
For each line we do not need a new try except clause. This can be included in the last one as well.